src/main/java/edu/ucsb/cs156/spring/backenddemo/services/PublicHolidayQueryService.java

```diff
@@ -27,7 +27,7 @@ public PublicHolidayQueryService(RestTemplateBuilder restTemplateBuilder) {
         restTemplate = restTemplateBuilder.build();
     }
 
-    public static final String ENDPOINT = "https://date.nager.at/api/v2/publicholidays/{year}/{countryCode}";
+    public static final String ENDPOINT = "https://date.nager.at/swagger/index.html";
```

Comment:
is this the correct link? The old one looked right

Reply:
![image](https://github.com/ucsb-cs156-s24/team01-s24-4pm-3/assets/42985687/0b2a8313-664d-4fe8-885d-d2569f1e1ab2)